src/stores/Frame/FrameStore.ts

```diff
@@ -110,6 +110,7 @@ export class FrameStore {
     public readonly colorbarStore: ColorbarStore;
 
     public spectralCoordsSupported: Map<string, {type: SpectralType; unit: SpectralUnit}>;
+    public secondarySpectralCoordsSupported: Map<string, {type: SpectralType; unit: SpectralUnit}>;
```

Comment:
perhaps we can remove `secondarySpectralCoordsSupported` and use `spectralCoordsSupported` for both